### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by darrachequesne**

### CODE REVIEW

**Comment by nkzawa** [comment]
:+1: awesome as alwasys

File: test/socket.io.js [hunk 1]
```diff
@@ -1511,7 +1511,7 @@ describe('socket.io', function(){
     it('should handle very large json', function(done){
       this.timeout(30000);
       var srv = http();
-      var sio = io(srv);
+      var sio = io(srv, { perMessageDeflate: false });
```
Comment (nkzawa) [inline test/socket.io.js]: Any reasons for this change?

Comment (darrachequesne) [inline test/socket.io.js]: Hi! Well, I think it causes some builds (like this one for example :angel:  https://travis-ci.org/socketio/socket.io/jobs/92141401) to pathetically fail (by timeout) due to the perMessageDeflate feature (which is somehow great, don't get me wrong!).

Comment (nkzawa) [inline test/socket.io.js]: reasonable

**Comment by darrachequesne** [comment]
Great, thanks!

**Pull request merged**